docs/index.Rmd

```diff
@@ -269,19 +273,23 @@ When you have completed labeling your image, it's time to export the labels you
 
   
 
-That's all! If you're ready to label another image, repeat from 'How to label: Part 1'. See the section 'Selecting a mission-date for the `eeplumB.js` script'. You do not need to wait for the task to complete to navigate away from this screen or to start a new script.
+That's all! You do not need to wait for the task to complete to navigate away from this screen or to start a new script.
+
+- If you were completing the validation workflow, check your labeled images against our expectations by ????. If you are happy with your results, you are ready to start labeling for real! Head to the `Selecting a mission-date for labeling` section to pick your own mission-date, and then start back at `How to label: Part 1`.
```

Comment:
https://code.earthengine.google.com/7dd1bca1b1a8f80dd51e045487db9a5a?hideCode=true
- If you were completing the validation workflow, check your labeled images against our expectations by heading to [this labeled validation set](https://code.earthengine.google.com/7dd1bca1b1a8f80dd51e045487db9a5a?hideCode=true). Click 'Run' and click through the Tile-Dates, comparing your labels to ours. Did you use the label classes similarly? Remember, it's less about how many pixels you've labeled and more that the 'quality' of the pixel label is high.

Reply:
```suggestion
- If you were completing the validation workflow, check your labeled images against our expectations by heading to [this labeled validation set](https://code.earthengine.google.com/7dd1bca1b1a8f80dd51e045487db9a5a?hideCode=true).  Click 'Run' and click through the Tile-Dates, comparing your labels to ours. Did you use the label classes similarly? Remember, it's less about how many pixels you've labeled and more that the 'quality' of the pixel label is high. If you are happy with your results, you are ready to start labeling for real! Head to the `Selecting a mission-date for labeling` section to pick your own mission-date, and then start back at `How to label: Part 1`.
```